metadata-connector/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "metadata_connector",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "clean": "rm -rf ./artifacts/ ./cache/",
+    "compile": "npx hardhat compile",
+    "test": "npx hardhat test",
+    "deploy:aurora": "yarn clean && yarn compile && npx hardhat run scripts/deploy.js --network aurora",
+    "deploy:ropsten": "yarn clean && yarn compile && npx hardhat run scripts/deploy.js --network ropsten"
```

Comment:
Please add `deploy:goerli` and `deploy:mainnet` commands

Reply:
removed this instead, added the function to deploy tokens from bridge-cli